include/cpuinfo.h

```diff
@@ -46,6 +46,10 @@
 	#endif
 #endif
 
+#if defined(__loongarch__) && defined(__loongarch64)
```

Comment:
Only matching against `__loongarch64` is enough, there's no configuration that has `__loongarch64` but is not LoongArch.

And the section below needs update too, you need a `#ifndef / #define CPUINFO_ARCH_LOONGARCH64 0` pair like all other arches.